scripts/fetch_articles_production.py

```diff
@@ -0,0 +1,372 @@
+"""
+The script is based on the API provided by europe PMC http://europepmc.org/
+ePMC treats the full text as free text, so when the accessions are mentioned in the full text, they will be indexed.
+Due to the factor that it is much more likely to mention the dataset accession as a whole than each individual records,
+e.g. paper PMC6500009 mentions several dataset accessions PRJNA526824 etc and the sample range SAMN11119414-SAMN11119461
+After retrieving dataset-article relationships, the same relationships will be extended to all samples/files under the
+dataset
+Using the example above, each individual sample between SAMN11119414-SAMN11119461 will have the article PMC6500009
+"""
+import os
+
+import requests
+import copy
+from elasticsearch import Elasticsearch
+from elasticsearch import RequestsHttpConnection
+from utils_fetch_articles import get_record_ids, get_record_details, insert_into_es
+from constants import STANDARD_FAANG
+from typing import Dict, Set, List
+
+SCRIPT_NAME = 'fetch_article'
+
+ARTICLE_MAPPING = {
+    'pmcId': 'pmcid',
+    'pubmedId': 'pmid',
+    'doi': 'doi',
+    'title': 'title',
+    'authorString': 'authorString',
+    'journal': 'journalTitle',
+    'issue': 'issue',
+    'volume': 'journalVolume',
+    'year': 'pubYear',
+    'pages': 'pageInfo',
+    'isOpenAccess': 'isOpenAccess',
+    'source': 'source'
+}
+ARTICLE_BASIC_FIELDS = {'title', 'year', 'journal', 'source'}
+
+# read from k8s secrets
+ES_USERNAME = os.getenv('es_user')
+ES_PASSWORD = os.getenv('es_password')
+
+ES_HOST = os.environ.get("ES_NODE")
+
+es = None
+
+
+def main():
+    """
+    Main function that will import publications for all entities
+    :return:
+    """
+
+    global es
```

Comment:
would be better to move it outside of main() function scope, so you don't need to use "global"

Reply:
es moved outside the main function